src/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,117 @@
+<!--
+  - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<docs>
+</docs>
+
+<template>
+	<progress class="progress-bar vue"
+		:class="{ 'warn': warn }"
+		:style="progressBarStyle"
+		:value="value"
+		max="100" />
+</template>
+
+<script>
+export default {
+
+	name: 'ProgressBar',
+
+	props: {
+		/**
+		 * An integer between 1 and 100
+		 */
+		value: {
+			type: Number,
+			default: 0,
+		},
+		/**
+		 * Determines the height of the progressbar
+		 * Possible values:
+		 * - 'small' (default)
+		 * - 'medium'
+		 */
+		size: {
+			type: String,
+			default: 'small',
+		},
+		/**
+		 * Applies an error color to the progressbar if true
+		 */
+		warn: {
+			type: Boolean,
+			default: false,
+		},
+	},
+
+	computed: {
+
+		progressBarStyle() {
+			let height = 0
+			let borderRadius = 0
+			if (this.size === 'small') {
+				height = '4px'
+				borderRadius = '2px'
+			} else if (this.size === 'medium') {
+				height = '6px'
+				borderRadius = '3px'
+			}
+			return {
+				height,
+				'border-radius': borderRadius,
```

Comment:
It should work with camelCase
```suggestion
				borderRadius,
```

Reply:
Nope, this is not working

Reply:
Ah, ok :cry: